app/rental_routes.py

```diff
@@ -0,0 +1,101 @@
+from flask import Blueprint, abort, jsonify, make_response, request
+
+from app import db
+from app.models.customer import Customer
+from app.models.rental import Rental
+from app.models.video import Video
+
+rentals_bp = Blueprint("rentals",__name__, url_prefix="/rentals")
+
+@rentals_bp.route("/<register>",methods=["POST"])
+def handle_rental(register):
+    request_body = request.get_json()
+    cust_id, vid_id = validates_request_body(request_body)
+    customer = Customer.query.get(cust_id)
+    video = Video.query.get(vid_id)
+    if register == "check-in":
+        rental = db.session.query(Rental).filter(Rental.customer_id == cust_id, Rental.video_id == vid_id).first()
+        if not rental:
+            return make_response({"message": f"No outstanding rentals for customer {cust_id} and video {vid_id}"}, 400)
+            
+
+        db.session.delete(rental)
+        customer.videos_checked_out_count -= 1
+        db.session.commit()
+        return make_response(jsonify(create_checkout_dict(video,customer)),200)
+
+    elif register == "check-out":
+        rental = Rental(
+            customer_id=customer.customer_id,
+            video_id=video.video_id
+        )
+        if video.gets_available_inventory() == 0:
+            return make_response({"message":"Could not perform checkout"}, 400)
+
+        customer.rentals.append(rental)
+        video.rentals.append(rental)
+        
+        customer.videos_checked_out_count += 1
```

Comment:
Just a note, that you can actually make a method to calculate the videos_checked_out_count by looking at the `rentals` relationship and counting the number that haven't been returned.  

Your solution totally works, but if other systems could operate on the database, it would be possible for the count to get out of sync with the number of checked out videos.

Something to think about.